shared/actions/login/index.js

```diff
@@ -343,6 +350,20 @@ export function logoutDone () : AsyncAction {
   }
 }
 
+export function clearStoredSecret (username: string) : AsyncAction {
+  return (dispatch, getState) => {
+    const params: login_clearStoredSecret_rpc = {
+      method: 'login.clearStoredSecret',
+      param: {
+        username
+      },
+      incomingCallMap: {},
+      callback: (error, response) => { console.log(error) }
```

Comment:
The type of this is:

```
callback: (null | (err: ?any) => void)
```

Meaning `callback` is null or there is a function that takes one maybe argument (`err`) and returns nothing.

So you don't need response as a second arg.

You also should only console log if there is an error i.e. `error && console.log(error)`